### PR TITLE
Feature/section 51 meeting and from

### PR DESCRIPTION
**Pull request by jonathangoulding**

### CODE REVIEW

File: packages/forms-web-app/src/pages/projects/section-51/utils/adviceVM.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const getAdviceName = ({ organisation, firstName, lastName }) => {
```
Comment (DavidBeale) [inline packages/forms-web-app/src/pages/projects/section-51/utils/adviceVM.js:1]: I'd prob say a ViewModel was not a util - but at the same level as controller etc

**Review by DavidBeale — APPROVED** [review]

File: packages/forms-web-app/src/pages/projects/section-51/utils/adviceVM.js (hunk 1 above)
Comment (jonathangoulding) [reply packages/forms-web-app/src/pages/projects/section-51/utils/adviceVM.js]: good shout. Have lifted it out

**Pull request merged**